config/initializers/lograge.rb

```diff
@@ -6,4 +6,5 @@
   config.lograge.enabled = true
   config.lograge.custom_options = ->(event) { Logs::LogBuilder.new(event).extra_logged_data }
   config.lograge.formatter = ->(data) { Logs::LogFormatter.new(data).call }
+  config.lograge.ignore_actions = ['HealthChecksController#index']
```

Comment:
We want to suppress these logs because this action will be requested a lot, and we don't want that much (mostly low value) clutter in our logs.